### PR TITLE
add script to change TMS api key for projects

### PR DESCRIPTION
**Pull request by Hagellach37**

### CODE REVIEW

File: mapswipe_workers/python_scripts/change_tms_api_key_for_projects.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        "The new api key will be obtained from the .env file. "
+        "Make sure to update this file first."
+    )
+
+    if tms_name in IMAGE_API_KEYS.keys():
+        new_api_key = IMAGE_API_KEYS[tms_name]
+        project_id_list = get_all_projects_by_tms(tms_name)
+        add_api_key_to_projects(project_id_list, new_api_key)
+    else:
+        logger.info(
```
Comment (laurentS) [inline mapswipe_workers/python_scripts/change_tms_api_key_for_projects.py:73]: Can you show this if the script is run without arguments? I feel like I'd want an easily accessible reminder from the makefile.

File: Makefile [hunk 2]
```diff
@@ -25,3 +25,6 @@ update_firebase_functions_and_db_rules:
 
 deploy_latest_workers_version:
 	git pull; docker compose build postgres django django-schedule-task manager_dashboard community_dashboard nginx api mapswipe_workers_creation mapswipe_workers_stats mapswipe_workers_firebase_to_postgres firebase_deploy; docker compose up -d postgres django django-schedule-task manager_dashboard community_dashboard nginx api mapswipe_workers_creation mapswipe_workers_stats mapswipe_workers_firebase_to_postgres firebase_deploy
+
+update_tms_api_key:
+	@echo "first update the .env file with the new api key and then run this script with tms_name=your_tms_provider_name to update API key"; docker-compose run --rm mapswipe_workers_creation python python_scripts/change_tms_api_key_for_projects.py $(tms_name)
```
Comment (laurentS) [inline Makefile:30]: What happens if run without argument? See my other comment below.

**Review by laurentS — COMMENTED** [review]
Nice!

**Review by laurentS — APPROVED** [review]

File: mapswipe_workers/python_scripts/change_tms_api_key_for_projects.py (hunk 1 above)
Comment (Hagellach37) [reply mapswipe_workers/python_scripts/change_tms_api_key_for_projects.py:78]: yes good catch. added this now

**Pull request merged**